docs/overview/software.md

```diff
@@ -37,10 +37,32 @@ The bridge is composed of three different pieces of software:
 - The `bridge` component inside [Witnet-Rust], which monitors the WBI
   for new requests and also scans
 
+## Open source libraries and development
+
+### Verifiable Random Function (VRFs) implementations
+
+- [vrf-solidity]: fast and effective library for VRFs written in Solidity
+- [vrf-rs]: implementation of Elliptic Curve VRFs written in Rust
+
+### Elliptic Curve libraries
```

Comment:
```suggestion
### Elliptic Curve
```

Reply:
Thank you @mariocao! That way it looks way cleaner!